packages/pwa-kit-runtime/package.json

```diff
@@ -46,19 +46,19 @@
   },
   "devDependencies": {
     "@loadable/component": "^5.15.3",
-    "@salesforce/pwa-kit-dev": "3.6.0-preview.0",
+    "@salesforce/pwa-kit-dev": "3.6.0-preview.1",
     "@serverless/event-mocks": "^1.1.1",
     "aws-lambda-mock-context": "^3.2.1",
     "fs-extra": "^11.1.1",
-    "internal-lib-build": "3.6.0-preview.0",
+    "internal-lib-build": "3.6.0-preview.1",
     "nock": "^13.3.0",
     "nodemon": "^2.0.22",
     "sinon": "^13.0.2",
     "superagent": "^6.1.0",
     "supertest": "^4.0.2"
   },
   "peerDependencies": {
-    "@salesforce/pwa-kit-dev": "3.6.0-preview.0"
```

Comment:
Note: this version was not updated previously 🤔 . We may want to keep an eye on this when we officially release it